src/config/lib/axios/index.ts

```diff
@@ -10,9 +10,13 @@
 import axios from 'axios';
 
 export const api = axios.create({
-  baseURL: 'https://alectrion-gateway-2023.herokuapp.com/',
+  baseURL: 'http://localhost:4000/',
```

Comment:
Favor voltar para url do deploy
```suggestion
  baseURL: 'https://alectrion-gateway-2023.herokuapp.com/',
```